### PR TITLE
Upgrade to Polymer 2.0

### PR DESCRIPTION
**Pull request by ronnyroeller**
Fixes https://github.com/PolymerElements/iron-swipeable-container/issues/25

### CODE REVIEW

**Comment by cdata** [comment]
Hey, thanks for the contribution! It looks like some details may need attending to before we can merge this.

File: bower.json [hunk 1]
```diff
@@ -21,15 +21,29 @@
   "homepage": "https://github.com/PolymerElements/iron-swipeable-container",
   "ignore": [],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.1.0"
+    "polymer": "Polymer/polymer#^2.0.0"
```
Comment (cdata) [inline bower.json]: In order for this element to work in "hybrid" mode, please use an inclusive version range that allows for 1.x compatible dependencies. Example: https://github.com/PolymerElements/paper-button/blob/master/bower.json#L24-L27

**Comment by notwaldorf** [comment]
@ronnyroeller are you still interested in finishing this PR, or should someone else (me) take it over?

**Comment by ronnyroeller** [comment]
@notwaldorf I'd appreciate if you could take it over. Thanks!

**Comment by notwaldorf** [comment]
Cool! Took your commit and opened https://github.com/PolymerElements/iron-swipeable-container/pull/29. Will close this so that we can merge that one (and you'll still get credit for the commit, don't worry)

**Pull request closed without merging**